liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -405,26 +400,32 @@ private static void parseIncludeVariables(Document doc) throws XPathExpressionEx
             }
         }
     }
-    
-    private static void parseConfigDropinsDirVariables(String inDir) throws XPathExpressionException, SAXException, IOException {
+
+    private static File getConfigDropinsDir() {
         File configDropins = null;
-        
-        // if configDirectory exists and contains configDropins directory, 
+
+        // if configDirectory exists and contains configDropins directory,
         // its configDropins has higher precedence.
         if (configDirectory != null && configDirectory.exists()) {
             configDropins = new File(configDirectory, "configDropins");
         }
-        
+
```

Comment:
If we can rely on the target for server.xml, then I think we should be able to rely on target for the configDropins directory as well. I think after we switch to target, then we don't need first `if`.